src/VerifierAlgorithm.ts

```diff
@@ -36,6 +36,15 @@ function extractPublicKeyBytes(pk: VerificationMethod): Uint8Array {
     return base64ToBytes((<LegacyVerificationMethod>pk).publicKeyBase64)
   } else if (pk.publicKeyHex) {
     return hexToBytes(pk.publicKeyHex)
+  } else if (pk.conditionWeightedThreshold && Array.isArray(pk.conditionWeightedThreshold)) {
+    return hexToBytes(
```

Comment:
nothin' to do here yet, but this needs to be recursive as the inner object may be a jwk, another type of verification method or even another verifiable condition

Reply:
added